package.json

```diff
@@ -37,7 +37,7 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "./node_modules/.bin/mocha --timeout 5000 -R spec",
+    "test": "./node_modules/.bin/mocha --timeout 9999 -R spec",
```

Comment:
I was comparing the changes of the master branch to this branch, and the timeout there was 9999.  And it seems to be working here.